segmentation/include/pcl/segmentation/impl/crf_segmentation.hpp

```diff
@@ -480,9 +480,6 @@ pcl::CrfSegmentation<PointT>::segmentPoints (pcl::PointCloud<pcl::PointXYZRGBL>
   // ----------------------------------//
   // --------      -------------------//
 
-  pcl::PointCloud<pcl::PointXYZRGBL> tmp_cloud;
-  tmp_cloud = *filtered_anno_;
```

Comment:
This is part of the commented code block which exists below. Maybe move this over there (line 549).